addons/sys_attenuate/CfgSoundEffects.hpp

```diff
@@ -1,49 +1,68 @@
-/* Use this to determine the attenution between people inside and outside vehicles. */
+/*
+ * acreAttenuation: Use this to determine the attenution between people inside and outside vehicles.
+ * acreAttenuationTurnedOut: Use this to determine the attenution between people inside (turned in) and turned out in a vehicle (refers to isTurnedOut command).
```

Comment:
If I understand correctly, this is when both players are in the vehicle, one turned in, one turned out. If either of them is outside the vehicle, `acreAttenuation` gets used instead? In that case, turned out is still treated like he was outside the vehicle?

Reply:
Yes, this is for when a player is turned in and one is turned out (previously this was hardcoded to be `0.5 * acreAttenuation`). No, `acreAttenuation` is used for turned in players and outside players. If a player is turned out and one is outside, there is no `acreAttenuation` (`acreAttenuation == 0`).

Reply:
Sorry, yes that's what I meant. Good, just wanted to confirm logic didn't change for that.